internal/tui/vuln-info.go

```diff
@@ -199,18 +205,20 @@ func (v *vulnInfo) graphOnlyView() string {
 	// and it's difficult to implement
 	s := strings.Builder{}
 	s.WriteString(vulnInfoHeadingStyle.Render("Affected:"))
-	var strs []string
+	strs := make([]string, 0, 2*len(v.chainGraphs))
```

Comment:
Is 2* correct here? If so, can you add a comment explaining why?

Reply:
2* is correct, added comment.